Geometry/GEMGeometryBuilder/src/GEMGeometryBuilderFromDDD.cc

```diff
@@ -118,10 +118,13 @@ GEMGeometry* GEMGeometryBuilderFromDDD::buildGeometry(DDFilteredView& fv, const
   // construct the regions, stations and rings. 
   for (int re = -1; re <= 1; re = re+2) {
     GEMRegion* region = new GEMRegion(re);
-    for (int st=1; st<=2; ++st) {
+    for (int st=1; st<=GEMDetId::maxStationId; ++st) {
       GEMStation* station = new GEMStation(re, st);
       std::string sign( re==-1 ? "-" : "");
       std::string name("GE" + sign + std::to_string(st) + "/1");
+      // Closest (furthest) super chambers in GE2/1 are called GE2/1s (GE2/1l)
+      if (st==2) name = "GE" + sign + std::to_string(st) + "/1s";
+      if (st==3) name = "GE" + sign + std::to_string(st-1) + "/1l";
```

Comment:
changes in this file revert changes from #15990.

Considering that this PR sets GEMDetId::maxStationId to 2, at least the last line looks irrelevant,
but then considering that long and short GE2/1s should have been gone with #15990, the last two line edits look wrong.